Makefile

```diff
@@ -17,18 +17,18 @@
 
 SHELL := /bin/bash -o pipefail
 
-VERSION ?= 0.1.0-SNAPSHOT
-RELEASE_SRC = apache-apisix-java-plugin-runner-${VERSION}-src
+VERSION ?= 0.1.0
+RELEASE_SRC = apisix-java-plugin-runner-${VERSION}
```

Comment:
why remove `-src`? other projects have it.

Reply:
got

Reply:
update